sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalog.scala

```diff
@@ -53,10 +53,15 @@ class V2SessionCatalog(catalog: SessionCatalog)
   override def listTables(namespace: Array[String]): Array[Identifier] = {
```

Comment:
There is no `listViews` in `TableCatalog` interface so I think it should be in `listTables`. IIUC existing v1 session catalog does this already (list views when calling `listTables`)